code/__HELPERS/patterns.dm

```diff
@@ -79,3 +79,38 @@
 				turfs_to_check += AdjT
 				results += AdjT
 	return results
+
+///Generates a cone shape. Any other checks should be handled with the resulting list. Can work with up to 359 degrees
+/proc/generate_cone(atom/center, outer_range = 10, inner_range = 1, cone_width = 60, cone_direction = 0, blocked = TRUE)
```

Comment:
Could you make this documentation more clear as to what the params do here?

Reply:
changed